packages/testkit/components/Modal.ts

```diff
@@ -20,13 +21,28 @@ export class Modal {
     this.page = page;
     this.locator = locator;
     this.elementReportName = elementReportName;
-    this.closeModalButton = new IconButton(this.page, this.locator, elementReportName);
+    this.closeModalButton = new IconButton(
+      this.page,
+      this.locator.getByTestId("close-modal-button"),
```

Comment:
change in here as well afterwards

Reply:
though we already talked about it, I strongly suggest that @vibe/testkit would match **@vibe/core only**. if it can match other custom usages by accident (e.g. other modals, etc.), then it's nice and useful, but everything under `@vibe` namespace, especially as we're open source, should match specifically our @vibe/core library.

also, we're planning on having testkit for unit tests in the future, and it should also match specifically for @vibe/core